man/systemd.netdev.xml

```diff
@@ -1979,6 +1982,77 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[Batadv] Section Options</title>
+    <para>The [Batadv] section only applies for
+    netdevs of kind <literal>batadv</literal> and accepts the
+    following keys:</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>GatewayMode=</varname></term>
+        <listitem>
+          <para>The GatewayMode is a string of either "off", "server" or "client"</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>BridgeLoopAvoidance=</varname></term>
+        <listitem>
+          <para>Enables (1) or disables (0) BridgeLoopAvoidance</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>DistributedArpTable=</varname></term>
+        <listitem>
+          <para>Enables (1) or disables (0) DistributedArpTable</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>Fragmentation=</varname></term>
+        <listitem>
+          <para>Enables (1) or disables (0) Fragmentation</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>HopPenalty=</varname></term>
+        <listitem>
+          <para>The hop penalty setting allows to modify batman-adv's preference for multihop
```

Comment:
batman-adv is the command? So this should be either `<command>batman-adv</command>` or `<citeref>...batman-adv...` if the man page is available somewhere.